src/veh_interact.cpp

```diff
@@ -663,13 +667,16 @@ void veh_interact::do_install()
                                                    part.has_flag(VPFLAG_CONTROLS) ||
                                                    part.location == "fuel_source" ||
                                                    part.location == "on_battery_mount" ||
-                                                   part.location.empty(); };
-    tab_filters[tab_filters.size()-1] = [&](const vpart_info *part) { // Other: everything that's not in the other filters
-        for (size_t i=1; i < tab_filters.size()-1; i++ ) {
+                                                   (part.location.empty() && part.has_flag("FUEL_TANK")); };
+
+    tab_filters[tab_filters.size()-2] = [&](const vpart_info *part) { // Other: everything that's not in the other filters
+        for (size_t i=1; i < tab_filters.size()-2; i++ ) {
             if( tab_filters[i](part) ) return false;
         }
         return true; };
-
+    tab_filters[7] = [&](const vpart_info *p){//The user specified filter
```

Comment:
Would be cleaner all on one line with the comment on the line above it. Sorry for all the corrections but its important the code style is consistent for the project